examples/layer2/vars/vars.yml

```
@@ -0,0 +1,18 @@
+# Equinix variables
+project_name: my_metal_layer2_project
+device_hostname: layer2-device
+metro: sv
+operating_system: ubuntu_20_04
+plan: c3.small.x86
+interconnection_name: "ansible-layer2-example"
+vrf_gateway_ip_range: 192.168.200.0/25
+vrf_peering_ip_network: 169.254.0.0
+vrf_peering_ip_range: "{{ vrf_peering_ip_network }}/29"
+vrf_vc_peering_ip_range: "{{ vrf_peering_ip_network }}/30"
+metal_peering_ip: "{{ vrf_vc_peering_ip_range | ansible.utils.nthhost(2) }}"
+aws_peering_ip: "{{ vrf_vc_peering_ip_range | ansible.utils.nthhost(1) }}"
+vrf_peering_asn: 65100
+fabric_connection_name: ansible-layer2-example
+# AWS variables
+aws_network_cidr: 172.16.0.0/16
```

```suggestion
metal_vrf_asn: 65000
```
Add a variable that represents the ASN of the Metal VRF.

---

Latest commit restructures the playbooks so that the correct AWS-side ASN is used, and the Metal-side ASN was moved to a var as part of those changes.